components/components/classnamesTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { googleCalendarAtomClassnamesPropData, availabilitySettingsAtomClassnamesPropData, bookerAtomPropsClassnamesPropData } from "@utils/atomsClassnameTableData";
+
+export default ({
+    atom
+  }: {
+    atom: string;
+  }) => {
+  let tableData = [{name: '', description: ''}];
+
+  console.log(atom, 'classnames table test');
+  
+
+  if (atom === "google calendar") { 
```

Comment:
atom names "google calendar", "availability settings" etc. can be stored in enum that is re-used, otherwise "google calendar" is copied in many places.

Reply:
Good catch, will fix this!